python/py-fastdtw/Portfile

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; truncate-lines: t; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:et:sw=4:ts=4:sts=4
+
+PortSystem              1.0
+PortGroup               python 1.0
+
+name                    py-fastdtw
+version                 0.3.4
+categories-append       science
+platforms               darwin
+license                 MIT
+maintainers             nomaintainer
+description             Dynamic Time Warping (DTW) algorithm with an O(N) time and memory complexity.
+long_description        Python implementation of FastDTW, which is an approximate Dynamic Time Warping (DTW) algorithm that provides optimal or near-optimal alignments with an O(N) time and memory complexity.
```

Comment:
Minor nitpick, but I would encourage folding this line using `\`

Reply:
Thank you for your review.
I have added `\` for every 80 characters, squashed the commit, and pushed as 3cf45cd .